doc/TUTORIAL.md

```diff
@@ -1,21 +1,18 @@
-Tutorial on how to add a badge for a service
-============================================
+# Tutorial on how to add a badge for a service
 
 This tutorial should help you add a service to shields.io in form of a badge.
 You will need to learn to use JavaScript, git and Github.
 Please [improve the tutorial](https://github.com/badges/shields/edit/master/doc/TUTORIAL.md) while you read it.
 
-(1) Reading
------------
+## (1) Reading
 
 You should read [CONTRIBUTING.md](../CONTRIBUTING.md)
 
 You can also read previous
 [merged pull-requests with the 'service-badge' label](https://github.com/badges/shields/pulls?utf8=%E2%9C%93&q=is%3Apr+label%3Aservice-badge+is%3Amerged)
 to see how other people implemented their badges.
 
-(2) Setup
----------
+## (2) Setup
```

Comment:
What would folks think about adding the url for the Github workflow to https://guides.github.com/activities/forking/ or changing the existing url? It is currently http://try.github.io/ which is a great resource for git

Reply:
K I'll plug these in shortly

Reply:
Gotcha, yeah I'll do that in a follow up (at some point)